projects/1.16/assets/corail-woodcutter/corail_woodcutter/lang/zh_cn.json

```
@@ -0,0 +1,59 @@
+{
+  "block.corail_woodcutter.oak_woodcutter": "橡木锯木机",
+  "block.corail_woodcutter.spruce_woodcutter": "云杉木锯木机",
+  "block.corail_woodcutter.birch_woodcutter": "白桦木锯木机",
+  "block.corail_woodcutter.jungle_woodcutter": "丛林木锯木机",
+  "block.corail_woodcutter.acacia_woodcutter": "金合欢木锯木机",
+  "block.corail_woodcutter.dark_oak_woodcutter": "深色橡木锯木机",
+  "block.corail_woodcutter.crimson_woodcutter": "绯红木锯木机",
+  "block.corail_woodcutter.warped_woodcutter": "诡异木锯木机",
+  "block.corail_woodcutter.cherry_woodcutter": "樱花木锯木机",
```

```suggestion
  "block.corail_woodcutter.cherry_woodcutter": "樱桃木锯木机",
```
这应该是BOP的联动吧，BOP的是樱桃木